### PR TITLE
EDU-12635 - Creating testing-the-server-side-rendering.md

### PR DESCRIPTION
**Pull request by barbara-celi**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

FastStore migration - [EDU-12635](https://vtex-dev.atlassian.net/browse/EDU-12635)

[EDU-12635]: https://vtex-dev.atlassian.net/browse/EDU-12635?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Testing the server-side rendering"
+tags: 
+    - performance
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md:4]: A ideia de usar as tags neste contexto seria útil se fosse um troubleshooting. Uma vez que o componente de renderização de tags estiver pronto, elas poderiam ser integradas aos cards de troubleshooting. No entanto, não tenho certeza se essa funcionalidade faz sentido para tutorias como este. Qual seria a ideia de passar as tags aqui?

File: docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+To ensure your server-side rendering (SSR) HTML is functional and free of bugs, it is important to check how your web page looks and behaves when JavaScript is disabled.
+
+## Step by step
+
+1. Check the documentation of your web browser and search how to disable JavaScript.
+
+    - If you are using Google Chrome, refer to Google's documentation on [Disabling JavaScript.](https://developers.google.com/web/tools/chrome-devtools/javascript/disable).
+
+2. Access your website and observe if its core functionalities are working as expected. Your store will not function exactly as planned but check below which are the expected and undesired behaviors when disabling JavaScript:
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md:17]:
```suggestion
2. Access your store website and observe if its core functionalities are working as expected. Your store will not function exactly as planned but check below which are the expected and undesired behaviors when disabling JavaScript:
```

File: docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    - performance
+---
+
+JavaScript handles data fetching, templating, and routing on the client-side rather than on the server-side. However, the amount of JavaScript shipped to the browser tends to increase as the complexity of an application grows. As a result, JavaScript code competes for processing power and delays the rendering task while striving to be processed before the browser finishes rendering the page content.
+
+To ensure your server-side rendering (SSR) HTML is functional and free of bugs, it is important to check how your web page looks and behaves when JavaScript is disabled.
+
+## Step by step
+
+1. Check the documentation of your web browser and search how to disable JavaScript.
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md:13]:
```suggestion
1. Check the documentation of your web browser and search for how to disable JavaScript.
```

File: docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+    - If you are using Google Chrome, refer to Google's documentation on [Disabling JavaScript.](https://developers.google.com/web/tools/chrome-devtools/javascript/disable).
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md:15]:
```suggestion
    - For example, if you are using Google Chrome, refer to Google's documentation on [Disabling JavaScript.](https://developers.google.com/web/tools/chrome-devtools/javascript/disable).
```

File: docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md (hunk 2 above)
Comment (Mariana-Caetano) [reply docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md:17]: > Access your store website and observe if its core functionalities are working as expected.

Talvez deixar mais claro o que seriam essas core functionalities, algo: "Access your store website and observe if its core functionalities are working as expected, such as product searching, shopping cart and checkout process."

File: docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+    - **Undesired behaviors**
+
+        - The page does not render anything.
+        - Instead of updating only the different content, the page is fully refreshed when navigating to another page.
+        - Misplaced layout.
+
+    - **Expected behaviors**
+
+        - Simpler design.
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md:27]: Talvez algo como "Basic layout similar to a static webpage".

**Review by Mariana-Caetano — CHANGES_REQUESTED** [review]

File: docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md (hunk 2 above)
Comment (barbara-celi) [reply docs/faststore/docs/managing-performance/testing-the-server-side-rendering.md]: @Mariana-Caetano vc acha que vale colocar um link pra página [VTEX Core Services](https://developers.vtex.com/docs/guides/getting-started#vtex-core-services)?

**Review by Mariana-Caetano — APPROVED** [review]
LGTM

**Pull request merged**